elasticgraph-indexer_autoscaler_lambda/elasticgraph-indexer_autoscaler_lambda.gemspec

```
@@ -16,6 +16,7 @@ ElasticGraphGemspecHelper.define_elasticgraph_gem(gemspec_file: __FILE__, catego
 
   spec.add_dependency "aws-sdk-lambda", "~> 1.125"
   spec.add_dependency "aws-sdk-sqs", "~> 1.80"
+  spec.add_dependency "aws-sdk-cloudwatch", "~> 1.10"
```

```suggestion
  spec.add_dependency "aws-sdk-cloudwatch", "~> 1.104"
```

1.10 was released in 2018. We should depend on the latest:

https://rubygems.org/gems/aws-sdk-cloudwatch/versions